### ISSUE TITLE
Bump html-to-text

### ISSUE THREAD

**Issue by ngryman** [jsonresume/resumeToText#5]
Hey,

Currently installing `resume-cli` with `yarn` leads to:
```sh
error html-to-text@0.0.8: The engine "node" is incompatible with this module. Expected version "~0.8.0".
error Found incompatible module
```

A more recent version of `html-to-text` relaxes the version constraint.

**Comment by rabeesh** [comment]
Yeah, the current npm version of `html-to-text` has dependency to old `"html-to-text": "0.0.8"`, npm owner has to bump this version and republish it.

**Comment by virgilwashere** [comment]
> Hey,
> 
> Currently installing `resume-cli` with `yarn` leads to:
> 
> ```shell
> error html-to-text@0.0.8: The engine "node" is incompatible with this module. Expected version "~0.8.0".
> ```

This should sort you out in the meantime.

```shell
yarn global add resume-cli --ignore-engines
```

Virgil
